joi-validations/users.validations.js

```diff
@@ -3,8 +3,7 @@ const {object, string} = require('./general.validations');
 const UserValidation = {
   authenticateWeb: {
     body: object({
-      country_code: string.max(2).min(2).uppercase().required(),
-      human_id: string.required()
+      exchangeToken: string.required()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from requiring `country_code` and `human_id` to just `exchangeToken` simplifies the authentication process. However, it's important to ensure that the `exchangeToken` is properly validated elsewhere in your code to prevent potential security issues. If not, you should consider adding additional validation here.
```diff
- exchangeToken: string.required()
+ exchangeToken: Joi.string().alphanum().min(3).max(30).required()
```
This will ensure that the `exchangeToken` is alphanumeric and between 3 and 30 characters long. Adjust the length according to your needs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->